Mathlib/Data/Fin/Basic.lean

```diff
@@ -346,86 +346,61 @@ theorem pos_iff_ne_zero' [NeZero n] (a : Fin n) : 0 < a ↔ a ≠ 0 := by
 
-theorem revPerm_involutive : Involutive (@revPerm n) :=
-  Involutive.toPerm_involutive _
-#align fin.rev_involutive Fin.revPerm_involutive
+theorem rev_injective : Injective (@rev n) :=
+  rev_involutive.injective
+#align fin.rev_injective Fin.rev_injectiveₓ
 
-theorem revPerm_injective : Injective (@revPerm n) :=
-  revPerm_involutive.injective
-#align fin.rev_injective Fin.revPerm_injective
+theorem rev_surjective : Surjective (@rev n) :=
+  rev_involutive.surjective
+#align fin.rev_surjective Fin.rev_surjectiveₓ
 
-theorem revPerm_surjective : Surjective (@revPerm n) :=
-  revPerm_involutive.surjective
-#align fin.rev_surjective Fin.revPerm_surjective
+theorem rev_bijective : Bijective (@rev n) :=
+  rev_involutive.bijective
+#align fin.rev_bijective Fin.rev_bijectiveₓ
 
-theorem revPerm_bijective : Bijective (@revPerm n) :=
-  revPerm_involutive.bijective
-#align fin.rev_bijective Fin.revPerm_bijective
+#align fin.rev_inj Fin.rev_injₓ
 
--- @[simp] -- Porting note: simp can prove this
-theorem revPerm_inj {i j : Fin n} : revPerm i = revPerm j ↔ i = j :=
-  revPerm_injective.eq_iff
-#align fin.rev_inj Fin.revPerm_inj
-
-@[simp]
-theorem revPerm_revPerm (i : Fin n) : revPerm (revPerm i) = i :=
-  revPerm_involutive _
-#align fin.rev_rev Fin.revPerm_revPerm
+#align fin.rev_rev Fin.rev_revₓ
 
 @[simp]
 theorem revPerm_symm : (@revPerm n).symm = revPerm :=
   rfl
 #align fin.rev_symm Fin.revPerm_symm
 
-theorem revPerm_eq {n a : ℕ} (i : Fin (n + 1)) (h : n = a + i) :
-    revPerm i = ⟨a, Nat.lt_succ_iff.mpr (Nat.le.intro h.symm)⟩ := by
-  ext
-  dsimp
-  conv_lhs =>
-    congr
-    rw [h]
-  rw [add_assoc, add_tsub_cancel_right]
-#align fin.rev_eq Fin.revPerm_eq
+#align fin.rev_eq Fin.rev_eqₓ
 
-@[simp]
-theorem revPerm_le_revPerm {i j : Fin n} : revPerm i ≤ revPerm j ↔ j ≤ i := by
-  simp only [le_iff_val_le_val, val_revPerm, tsub_le_tsub_iff_left (Nat.add_one_le_iff.2 j.is_lt),
-    add_le_add_iff_right, iff_self]
-#align fin.rev_le_rev Fin.revPerm_le_revPerm
+#align fin.rev_le_rev Fin.rev_le_revₓ
 
-@[simp]
-theorem revPerm_lt_revPerm {i j : Fin n} : revPerm i < revPerm j ↔ j < i :=
-  lt_iff_lt_of_le_iff_le revPerm_le_revPerm
-#align fin.rev_lt_rev Fin.revPerm_lt_revPerm
+#align fin.rev_lt_rev Fin.rev_lt_revₓ
 
 /-- `Fin.rev n` as an order-reversing isomorphism. -/
 @[simps! apply toEquiv]
 def revOrderIso {n} : (Fin n)ᵒᵈ ≃o Fin n :=
-  ⟨OrderDual.ofDual.trans revPerm, revPerm_le_revPerm⟩
+  ⟨OrderDual.ofDual.trans revPerm, rev_le_rev⟩
 #align fin.rev_order_iso Fin.revOrderIso
 #align fin.rev_order_iso_apply Fin.revOrderIso_apply
 #align fin.rev_order_iso_to_equiv Fin.revOrderIso_toEquiv
 
 @[simp]
-theorem revOrderIso_symm_apply (i : Fin n) : revOrderIso.symm i = OrderDual.toDual (revPerm i) :=
+theorem revOrderIso_symm_apply (i : Fin n) : revOrderIso.symm i = OrderDual.toDual (rev i) :=
   rfl
-#align fin.rev_order_iso_symm_apply Fin.revOrderIso_symm_apply
+#align fin.rev_order_iso_symm_apply Fin.revOrderIso_symm_applyₓ
```

Comment:
Please remove *all* the `\_x`s. They are not helpful.

Reply:
@digama0, do you agree with this?